doc/model/train-se-atten.md

```diff
@@ -0,0 +1,122 @@
+```json
+"type_embedding":{
+            "neuron":           [8],
+            "resnet_dt":        false,
+            "seed":             1
+        }
+```
+You can add these settings in input.json if you want to change the default ones, see [here](train-se-e2-a-tebd.md) for detail information.
+
+
+### Type map
+For training a large systems, especially those with dozens of elements, the {ref}`type <model/type_map>` determines the element index of training data:
+```json
+"type_map": [
+   "Mg",
+   "Al",
+   "Cu"
+  ]
+```
+which should include all the elements in the dataset you want to train on. 
+## Data format
+DPA-1 supports the standard data format, which is detailed in [data-conv.md](../data/data-conv.md) and [system.md](../data/system.md).
+Note that in this format, only those frames with the same fingerprint(i.e. the number of atoms of different element) can be put together as a unit system.
+This may lead to sparse frame number in those rare systems. 
+
+An ideal way is to put systems with same total number of atoms together, which is the way we trained DPA-1 on [OC2M](https://github.com/Open-Catalyst-Project/ocp/blob/main/DATASET.md). 
+This system format, which is called `mixed_type`, is proper to put frame-sparse systems together and is slightly different from the standard one.
+Take an example, a `mixed_type` may contain the following files:
+```
+type.raw
+type_map.raw
+set.000/box.npy
+set.000/coord.npy
+set.000/energy.npy
+set.000/force.npy
+set.000/real_atom_numbs.npy
+set.000/real_atom_types.npy
+```
+This system contains `Nframes` frames with the same atom number `Natoms`, the total number of element types contained in all frames is `Ntypes`. Note that we put all the frames in one set `set.000`. Most files are the same as those in [standard format](../data/system.md), here we only list the distinct ones:
+
+ID         | Property                         | File                | Required/Optional    | Shape                    | Description
+---------- | -------------------------------- | ------------------- | -------------------- | -----------------------  | -----------
+fake_type  | Atom type indexes (place holder) | type.raw            | Required             | Natoms                   | All zeros to fake the type input
+type_map   | Atom type names                  | type_map.raw        | Required             | Ntypes                   | Atom names that map to atom type contained in all the frames, which is unnecessart to be contained in the periodic table
+real_type  | Atom type indexes of each frame  | real_atom_types.npy | Required             | Nframes \* Natoms        | Integers that describe atom types in each frame, corresponding to indexes in type_map
+type_number| Number of atoms in each type     | real_atom_numbs.npy | Required             | Nframes \* Ntypes        | Integers that describe the totoal number of atoms with the same type in each frame
```

Comment:
I do not see where do you implement the IDs `fake_type`, `real_type` and `type_number`. The ID means the keys in the data_dict of DeepmdData.